.github/workflows/tests.yml

```diff
@@ -12,12 +12,17 @@ jobs:
           '3.11',
           '3.12',
         ]
-    name: Python ${{ matrix.python-version }}
+        pip-override: [
+          '',
+          'pip install "urllib3<2"',
```

Comment:
You don't need to test all versions with that? Maybe just 3.9?

Reply:
That's what I had at the beginning but I figured that it's not only related to Python 3.9 at this point, it's more about people building a Python version with an older OpenSSL